Lesson_2_Python_II/Lesson_2.ipynb

```diff
@@ -2703,7 +2703,7 @@
     "baseball = pd.read_csv('baseball_data.csv', index_col = 0)\n",
```

Comment:
<p>fix the wording about "players that are 73 in tall" to "taller than 73"</p><p>Same with last print statement</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theLeague-AI/TheLeagueAI_main/pull/24/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='84'/>